ejemplo_langgraph_avanzado.py

```
@@ -0,0 +1,391 @@
+    Realiza operaciones matemáticas simples.
+    
+    Args:
+        operacion: Expresión matemática como string (ej: "2 + 2", "10 * 5")
+        
+    Returns:
+        Resultado de la operación
+    """
+    try:
+        # ADVERTENCIA: eval() es inseguro en producción
+        # Usar solo para propósitos educativos
+        resultado = eval(operacion)
```

Ruff (0.13.3)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)